main.go

```diff
@@ -19,7 +19,8 @@ var Version = "dev"
 
 // Options contains the flag options
 type Options struct {
-	Web3Endpoint string `long:"rpc" description:"Ethereum JSONRPC provider, such as Infura or Cloudflare" default:"https://mainnet.infura.io/v3/af500e495f2d4e7cbcae36d0bfa66bcb"` // Versus API key on Infura
+	Methods      map[string]int64 `short:"m" long:"method" description:"A map from json rpc methods to their weight" default:"eth_getCode:100" default:"eth_getLogs:250" default:"eth_getTransactionByHash:250" default:"eth_blockNumber:350" default:"eth_getTransactionCount:400" default:"eth_getBlockByNumber:400" default:"eth_getBalance:550" default:"eth_getTransactionReceipt:600" default:"eth_call:2000"`
```

Comment:
It's a bit unfortunate that these defaults are buried in a single long tag line here, but not sure I have a much better solution right now.

Reply:
I agree.
Another option would be to set it in the code later on, but that prevents these from being shown with `--help`.

Reply:
Agree. Good enough for now, let's see if we come up with something better later. :)